Formula/liquidprompt.rb

```diff
@@ -3,10 +3,19 @@ class Liquidprompt < Formula
   homepage "https://github.com/nojhan/liquidprompt"
   url "https://github.com/nojhan/liquidprompt/archive/v_1.10.tar.gz"
   sha256 "5a1e01b86e81002ac20c61cbd96f5fd276db7f3fa09ef220f3db0fb7aa614e24"
-  head "https://github.com/nojhan/liquidprompt.git", :branch => "develop"
 
   bottle :unneeded
 
+  head do
+    url "https://github.com/nojhan/liquidprompt.git", :branch => "master"
+    version "head"
+  end
+
+  devel do
```

Comment:
We don't accept new `devel` blocks that aren't pinned to a tarball or specific git tag/revision. We plan to eventually bottle `devel` stuff and we can't do that if they can move around freely.